### ISSUE TITLE
I cant get the results which are showed in the article.

### ISSUE THREAD

**Issue by rophlly** [ashisht96/DeepPS2#3]
1. I follow the right steps in README, but the MAE of 'reading' object is nearly 6 which is far below 18.34 mentioned in the article.
2. The paper mentioned and used normals and light directions as weak supervision, but in the code, in the first 25 epoch, normal and light directions are served as loss functions, why? It`s not consistent with what is mentioned in the article.
3. I didt find any switch of calibrated or uncalibrated experiment settings, but in the article, you compared DeepPS2 in both calibrated and uncalibrated conditions, what is the difference? And how to train the model with no light directions?
